lib/archive/util.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package archive
+
+import (
+	"archive/tar"
+	"bytes"
+)
+
+// Below are some structs and utility functions for unit testing the untar functionality
```

Comment:
Originally they were in a file that has a `_test` suffix. I intend to remove the code that is there and reference this function in two different test files. I will look to move it to `Unpack_test.go` the other place they are referenced is in image_test.go for the vsphere image_store I believe.